tests/test_expressions.py

```diff
@@ -520,7 +520,6 @@ def remove_column_b(node):
             return node
 
         self.assertEqual(expression.transform(remove_column_b).sql(), "SELECT a FROM x")
-        self.assertEqual(expression.transform(lambda _: None), None)
```

Comment:
Is this not true anymore?

Reply:
right because we don't allow transform to return None